staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -53,6 +55,12 @@ const (
 	// 34 chose as a number close to 30 that is likely to be unique enough to jump out at me the next time I see a timeout.
 	// Everyone chooses 30.
 	requestTimeout = 34 * time.Second
+	// DuplicateOwnerReferencesWarningFormat is the warning that a client receives when a create/update request contains
+	// duplicate owner reference entries. We don't provide details about all the duplicates in the request, to prevent the
+	// warning message from getting truncated. We do provide the UID of the last duplicate entry we seen as a hint, to
+	// help the client identify and fix the request.
+	// TODO(roycaihw): stop serving requests with duplicate owner reference entries in 1.24
+	DuplicateOwnerReferencesWarningFormat = ".metadata.ownerReferences contains duplicate entries, last duplicate UID: %v; API server dedups owner references in 1.20+, rejects such requests in 1.24+; please fix your requests"
```

Comment:
rather than promise us to a future release, perhaps, "may reject such requests as early as 1.24"?

Reply:
done